Github issues templates (#4386)
organicmaps/organicmaps: .github/ISSUE_TEMPLATE/feature_request.md

---
name: Feature request
about: Suggest an idea for OrganicMaps
title: ''

> labels: ''

Reviewer: ```suggestion
labels: [Enhancement]
```
Is it the correct format?

assignees: ''

---

**Is your feature request related to a problem? Please describe.**

> A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]

Reviewer: ```suggestion
A clear and concise description of what the problem is. For example:
I'm always frustrated when [...]
```

> **Describe the solution you'd like**

Reviewer: ```suggestion
**Describe the solution you would like**
```

> A clear and concise description of what you want to happen.

Reviewer: ```suggestion
A clear and concise description of what you want to see in Organic Maps.
```

> **Describe alternatives you've considered**

Reviewer: ```suggestion
**Describe alternatives you have considered**
```

> A clear and concise description of any alternative solutions or features you've considered.

Reviewer: ```suggestion
- How do you solve this issue now with Organic Maps or other apps?
- Attach any examples, screenshots, or screen recordings from other apps that help us to better understand the idea.
```
